llvm/test/CodeGen/AMDGPU/attributor-noalias-addrspace-parse.mir

```diff
@@ -0,0 +1,44 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
MIR print and parse tests belong in test/CodeGen/MIR, also remove attributor from the name. It's unrelated